apps/judicial-system/web/src/routes/Prosecutor/components/PoliceCaseFiles/PoliceCaseFiles.tsx

```diff
@@ -151,7 +152,7 @@ const PoliceCaseFiles: React.FC<Props> = ({
         files: policeData ? policeData.policeCaseFiles : [],
         isLoading: false,
         hasError: true,
-        errorCode: policeDataError?.graphQLErrors[0].extensions?.code,
+        errorCode: policeDataError?.graphQLErrors[0].extensions.code as string,
```

Comment:
I think adding a few `?.` should do the trick here. `extensions` if defined is always an object/map https://spec.graphql.org/draft/#sel-FAPHRPDZAADbBAzwM
```suggestion
        errorCode: policeDataError?.graphQLErrors[0]?.extensions?.code as string,
```